src/TableView/TableActions.js

```diff
@@ -308,6 +309,16 @@ function deleteError(error) {
   };
 }
 
+function _previousPageIfEmptyAfterDeletion(numberDeleted, state, plural, dispatch) {
```

Comment:
Please change signature of this function, you don't need add underscore before name.